.github/workflows/base.yml

```diff
@@ -77,6 +83,8 @@ env:
   QUARKUS_PATH: ${{ github.workspace }}/quarkus
   MANDREL_IT_PATH: ${{ github.workspace }}/mandrel-integration-tests
   MANDREL_PACKAGING_REPO: ${{ github.workspace }}/mandrel-packaging
+  BUILD_STATS_TAG: gha-22.3.0-dev-test-nightly
```

Comment:
Do we want to use the same tag for all nightly builds? Should we tag them differently depending on commit hash or run?

Reply:
I haven't thought about it yet :-) It may make sense to keep nightly runs in similar tags. I'm thinking `gha-22.3-dev-nightly-jdk11` and `gha-22.3-dev-nightly-jdk17` perhaps. Thoughts? Maybe we should start supporting a list of tags (for uploads of stats)? But we are getting ahead of ourselves here. I'll be sure to provide this input when run for nightly. This was meant as a catch-all. If nothing has been specified. Better than a null tag. I'll fix it.

Reply:
Updated patch sets the tag for nightly invocations. I think we want to have a single tag for each OS, Graal VM version, JDK version, quarkus version quadruplets so that we collect better long time stats. What do you think?

Reply:
OK. Keep in mind that we don't need to use tags. It's an **additional** thing which allows us to retrieve a set of stats. When querying you can safely ignore it. Feel free to have a look at https://github.com/Karm/collector/ as that is the thing collecting those stats. Right now, we don't collect `OS`, `commit-hash` or `quarkus-version`, it can get added later, though. Adding the quarkus version to the tag is one way to collect this info now. This is an **initial** patch which allows us to collect *some*  stats. We are free to change this scheme to whatever we like. Having tags on top of everything else we need, does no harm, IMHO.

Reply:
ACK, feel free to proceed once all the dependencies are in place.

Reply:
OK.